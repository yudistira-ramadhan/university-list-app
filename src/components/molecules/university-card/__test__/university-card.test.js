import React from 'react'
import ReactDOM from 'react-dom'
import UniversityCard from '../university-card'
import { render } from '@testing-library/react'
import renderer from 'react-test-renderer'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<UniversityCard />, div)
})

it('renders university card correctly', () => {
  const { getByTestId } = render(<UniversityCard />)
  expect(getByTestId('university-card')).toBeTruthy()
})

it('matches snapshot', () => {
  const tree = renderer.create(<UniversityCard />).toJSON()
  expect(tree).toMatchSnapshot()
})