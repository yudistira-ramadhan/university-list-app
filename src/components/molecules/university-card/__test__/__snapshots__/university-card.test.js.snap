// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="text-center card"
  data-testid="university-card"
>
  <div
    className="card-header"
  >
    Default Name
  </div>
  <div
    className="card-body"
  >
    <div
      className="card-title"
    >
      Default Country
    </div>
    <hr
      className="my-2"
    />
    <p
      className="card-text"
    >
      <a
        href={Array []}
        rel="noopener"
        target="blank"
      />
    </p>
  </div>
</div>
`;
